CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
-## 0.14.8-dev3
+## 0.14.8-dev4
```

Comment:
Let's bump to `0.15.0` when we release this since it's a major change.

Reply:
yep, will do.